sandbox.sh

```diff
@@ -0,0 +1,205 @@
+#! /bin/bash
+
+set -e
+
+RPC_NODE=http://localhost:20000
+
+# This secret key never changes.
+SECRET_KEY="edsk3QoqBuvdamxouPhin7swCvkQNgq4jP5KZPbwWNnwdZpSpJiEbq"
+
+DATA_DIRECTORY="data"
+
+SIDECLI=$(esy x which sidecli)
+sidecli() {
+  eval $SIDECLI '"$@"'
+}
+
+tezos-client() {
+  docker exec -it my-sandbox tezos-client "$@"
+}
+
+ligo() {
+  docker run --rm -v "$PWD":"$PWD" -w "$PWD" ligolang/ligo:0.28.0 "$@"
+}
+
+VALIDATORS=(0 1 2)
+
+message() {
+  echo "=========== $@ ==========="
+}
+
+validators_json() {
+  ## most of the noise in this function is because of indentation
+  echo "["
+  for VALIDATOR in "${VALIDATORS[@]}"; do
+    i=$(echo $VALIDATOR | awk -F';' '{ print $1 }')
+    ADDRESS=$(echo $VALIDATOR | awk -F';' '{ print $4 }')
+    URI=$(echo $VALIDATOR | awk -F';' '{ print $3 }')
+    if [ $i != 0 ]; then
+      printf ",
+"
+    fi
+    cat <<EOF
+  {
+    "address": "$ADDRESS",
+    "uri": "$URI"
+EOF
+    printf "  }"
+  done
+  echo ""
+  echo "]"
+}
+
+trusted_validator_membership_change_json() {
+  echo "["
+  for VALIDATOR in "${VALIDATORS[@]}"; do
+    i=$(echo $VALIDATOR | awk -F';' '{ print $1 }')
+    ADDRESS=$(echo $VALIDATOR | awk -F';' '{ print $4 }')
+    if [ $i != 0 ]; then
+      printf ",
+"
+    fi
+    cat <<EOF
+  {
+    "action": [ "Add" ],
+    "address": "$ADDRESS"
+EOF
+    printf "  }"
+  done
+  echo ""
+  echo "]"
+}
+
+create_new_validators() {
```

Comment:
This is not only creating new validators, it setup's the entire deku environment